templates/org/aac/aac-example-workload-content.md

```diff
@@ -1,40 +1,42 @@
-<!-- Use the aac-browse-header.yml   -->
+<!--Add the aac-browse-header.yml here.-->
```

Comment:
Do you think readers will know where the aac-browse-header.yml file is located? Can we provide a clearer instruction?